utils/issue-crendentials.js

```diff
@@ -0,0 +1,31 @@
+import { createRegistry } from './dock-registries';
+import { createCredential, distributeCredential } from './dock-credentials';
+import { waitForJobCompletion } from './dock-jobs';
+import { toast } from 'sonner';
+
+export const issueRevokableCredential = async (credential, setRevokableCredential) => {
+
+    const credentialPayload = credential;
+    const _credential = credentialPayload.credential;
+    const type = 'DockVBAccumulator2022';
+    //CREATING REGISTRY
+    const registry = await createRegistry(_credential.issuer.id, type);
+    //WAITING FOR REGISTRY JOB CONFIRMATION
+    await waitForJobCompletion(registry.id);
+    //SIGNING CREDENTIAL
+    const signed = await createCredential(registry.data.id, credentialPayload);
+
+    const issuedCredential = await distributeCredential(signed.data);
```

Comment:
Credentials can be distributed if you add `"distribute": true` to the `POST /credentials` body so this is not needed. (Only if we don't want to showcase how credentials can be manually distributed)

Reply:
That's correct! I removed distribute:true from _credentials